nodejs.spec

```diff
@@ -116,7 +122,11 @@ Release: %{npm_release}%{?dist}
 
 Obsoletes: npm < 0:3.5.4-6
 Provides: npm = %{npm_epoch}:%{npm_version}
+%if 0%{?rhel} > 7
+Requires: nodejs = %{epoch}:%{nodejs_version}-%{nodejs_release}%{?dist}
+%else
 Requires: rhoar-nodejs = %{epoch}:%{nodejs_version}-%{nodejs_release}%{?dist}
+%endif
```

Comment:
Alternatively, this entire block could just be replaced with the following w/o the rhel conditional:

`Requires: %{name} = %{epoch}:%{nodejs_version}-%{nodejs_release}%{?dist}`